docs/tutorial/part-five/index.md

```diff
@@ -75,42 +75,42 @@ module.exports = {
 }
 ```
 
-Save that and restart the gatsby development server. Then open up GraphiQL again.
+सहेजें और gatsby विकास सर्वर को पुनरारंभ करें। इसके बाद फिर से GraphiQL को खोलें।
 
-In the explorer pane, you'll see `allFile` and `file` available as selections:
+एक्सप्लोरर फलक में, आप `allFile` और` file` चयनों के रूप में उपलब्ध देखेंगे:
 
 ![graphiql-filesystem](graphiql-filesystem.png)
 
-Click the `allFile` dropdown. Position your cursor after `allFile` in the query area, and then type <kbd>Ctrl + Enter</kbd>. This will pre-fill a query for the `id` of each file. Press "Play" to run the query:
+`AllFile` ड्रॉपडाउन पर क्लिक करें। क्वेरी क्षेत्र में `allFile 'के बाद अपना कर्सर रखें, और फिर टाइप करें <kbd> Ctrl + Enter </ kbd>। यह प्रत्येक फ़ाइल के `आईडी` के लिए एक क्वेरी भर देगा। क्वेरी चलाने के लिए "Play" दबाएं:
```

Comment:
`<kbd> Ctrl + Enter </ kbd>`
=>
`<kbd>Ctrl + Enter</kbd>`

Reply:
Can you recheck this, you have missed an extra space in closing tag  here `< /kbd>`